docusaurus.config.js

```diff
@@ -150,11 +150,8 @@ module.exports = {
       '@docusaurus/preset-classic',
       {
         docs: {
-          path: 'gosling-docs/docs',
+          path: 'docs',
```

Comment:
- Can we entirely remove the `gosling-docs` folder in the repository?
- Should we archive `gosling-lang/gosling-docs` since I expect we will be maintaining docs in this repo directly?